genome_designer/utils/data_export_util.py

```diff
@@ -171,6 +171,9 @@ def export_contig_list_as_vcf(contig_list, vcf_dest_path_or_filehandle):
             cassette_sequence = str(contig_seqrecord.seq[
                     contig_left:contig_right])
 
+        if contig_left > contig_right:
```

Comment:
This code should be immediately after you parse `contig_left` and `contig_right`